### DISCUSSION TITLE
关于vc-primitive数据响应的问题

### DISCUSSION THREAD

**Discussion by MangMax** [zouyaoji/vue-cesium#361] [Q&A]
[demo](https://codepen.io/1061875478/pen/dyqMNQb)
[官网例子](https://zouyaoji.top/vue-cesium/#/zh-CN/component/primitives/vc-primitive)
使用了官网的一个例子，可以看到数据填装以后vc-primitive的绑定数据并没有改变。
我在实际开发中也遇到过，目前只能reload才能实现数据改变，不知道是哪里使用有问题

**Comment by zouyaoji** [comment]
你好说具体点呢。没看明白  “绑定数据并没有改变”

**Reply by MangMax** [reply]
![image](https://user-images.githubusercontent.com/68525017/220534930-319493a4-1513-4b1c-b6d6-b4aa572100ff.png)
这个是官网的例子，圆形右边的方形是正常加载的
![image](https://user-images.githubusercontent.com/68525017/220535086-7ec95c78-cc5e-44b7-b9c1-6f4589230c88.png)
在这里方形就没有加载

代码我看了一遍是一样的，是我哪里漏了吗

**Reply by zouyaoji** [reply]
cdn里面对大小写不敏感，无法识别驼峰写法。改成短横线写法即可。

:geometry-instances="geometryInstances"

**Reply by MangMax** [reply]
> cdn里面对大小写不敏感，无法识别驼峰写法。改成短横线写法即可。
> 
> :geometry-instances="geometryInstances"

是我理解错了，我的目的是想更新primitive，但是GeometryInstance是只读对象，我现在只能通过重载组件实现更新，但是会有闪烁的问题，是不是应该让vc-primitive支持GeometryVisualizer来实现更新呢
